Sources/StreamChatUI/ChatMessageList/ChatMessageListView.swift

```diff
@@ -147,14 +147,13 @@ open class ChatMessageListView: UITableView, Customizable, ComponentsProvider {
         let lastMessageIndexPath = IndexPath(row: 0, section: 0)
         let prevMessageIndexPath = IndexPath(row: 1, section: 0)
 
-        if
-            rectForRow(at: prevMessageIndexPath).minY < contentOffset.y,
-            rowsRange.contains(prevMessageIndexPath.row) {
-            scrollToRow(at: prevMessageIndexPath, at: .top, animated: false)
+        if rectForRow(at: prevMessageIndexPath).minY < contentOffset.y,
+           rowsRange.contains(prevMessageIndexPath.row) {
+            scrollToRow(at: prevMessageIndexPath, at: .bottom, animated: animated)
```

Comment:
The cell is inverted over Y axis so we should scroll to `.top`. It's confusing, I now 😄

Reply:
With `.top` the scroll animation was a bit weird actually, or maybe was the lack of the animation 🤔 But either way, it doesn't seem to have a difference 🤔